packages/css-library/tokens/fonts.json

```diff
@@ -43,6 +43,7 @@
       "heading": { "value": "1.2" }
     },
     "size": {
+      "source-sans-normalized": { "value": "1.06rem" },
```

Comment:
@powellkerry I looked back at [our original slack conversation](https://dsva.slack.com/archives/G01BJ3ESXL4/p1731094289138849?thread_ts=1731082541.131579&cid=G01BJ3ESXL4). 

Currently in this PR we're just generating a `font-size-root` token, given your changes above on line 21. If we want to additionally add a `vads-font-size-root` token, then another entry needs to be added here in the `vads-font.size` object, similar to what you've added above for `source-sans-normalized`. Adding a `"root": {"value": "1rem"}` entry here should generate a token named `vads-font-size-root`.